packages/data/src/redux-store/metadata/selectors.js

```diff
@@ -153,3 +158,33 @@ export function hasResolvingSelectors( state ) {
 		)
 	);
 }
+
+/**
+ * Retrieves the total number of selectors, grouped per status.
+ *
+ * @param {State} state Data state.
+ *
+ * @return {Object} Object, containing selector totals by status.
+ */
+export const countSelectorsByStatus = createSelector( ( state ) => {
```

Comment:
Is it possible to make this a private selector? It's a rather obscure, single-purpose API.

Reply:
I don't see a good reason to do that. While it's a single-purpose selector, I think about it as the rest of the metadata selectors and can definitely find a utility for it outside of the single suggested use in #53399. Perhaps we'll use it to build other loading progress bars, and perhaps plugin authors could want to do the same. 

Ideally, I'd like to follow up and improve the way we're accessing the internal `EquivalentKeyMap` map, but that's work for another PR and will require changes to the `equivalent-key-map` package.